src/components/ActNowPage/LoggedIn.js

```diff
@@ -1,17 +1,43 @@
 import { Box, Button } from "@material-ui/core";
 import clsx from "clsx";
-import { signOut } from "next-auth/react";
+import { signOut, useSession } from "next-auth/react";
 import Image from "next/image";
 import PropTypes from "prop-types";
 import React from "react";
 
 import useStyles from "./useStyles";
 
-import actNowLogo from "@/promisetracker/assets/Component 121 – 1@2x.png";
+import actNowLogo from "@/promisetracker/assets/actNowLogo2x.png";
 import ContentPage from "@/promisetracker/components/ContentPage";
+import Section from "@/promisetracker/components/ContentPage/Section";
+import Petitions from "@/promisetracker/components/Petitions";
+import Tabs from "@/promisetracker/components/Tabs";
 
-function ActNowLoggedInPage({ footer, title, navigation, ...props }) {
+function ActNowLoggedInPage({
+  footer,
+  title,
+  navigation,
+  petitions,
+  ...props
+}) {
   const classes = useStyles(props);
+  const { data: session } = useSession();
+  const started = petitions?.filter(
+    (petition) => petition.owner.user.email === session.user.email
+  );
+  const signed = petitions?.filter((petition) =>
+    petition.signatures.find(({ id }) => id === session.user.id)
+  );
```

Comment:
What if there are 10s of thousands of petitions? Are you going to load all of them and filter here?

1. Use `owner` query param to get the list of petitions you started e.g. `https://actnow.dev.codeforafrica.org/v1/petitions/?owner=3`
2. User `followers` query param to get the list of petitions you signed e.g. `https://actnow.dev.codeforafrica.org/v1/petitions/?followers=3`

I'm not 💯 sure on the `followers` though, there could be a difference between _following_ and _signing_ but either way, the API should have a simple way of getting all petitions a user has signed.

Reply:
thanks, @kilemensi . I think this works, I was having a conversation with @esirK about it yesterday. So I think the only thing left is to return the user Profile Id when signed in @esirK So i can use it to make the query